packages/debug/src/browser/view/debug-configuration-select.tsx

```diff
@@ -103,7 +103,7 @@ export class DebugConfigurationSelect extends React.Component<DebugConfiguration
         if (!value) {
             return false;
         } else if (value === DebugConfigurationSelect.ADD_CONFIGURATION) {
-            this.manager.addConfiguration();
+            setTimeout(() => this.manager.addConfiguration());
```

Comment:
I agree that this is a bit 'hacky' and I tested `animationFrame().then(() => this.manager.addConfiguration());` which also seems to work but I don't see any additional benefit from using that instead. I also did not find any alternative solution that solves that issue but I also don't think there are going to be any side effects from this, so in my opinion this could be merged.

The only thing that would be nice is a short comment explaining why this particular call needs to be handled differently though in theory that could be derived from the Git history. @tsmaeder How do you usually handle these situations with regard to documentation?